### DISCUSSION TITLE
Render quality (720p, 1080p)

### DISCUSSION THREAD

**Discussion by michielst** [inlife/nexrender#606] [Q&A] [answered]
Hey everyone,

I'm trying to use nexrender to create quick previews videos. I haven't found out how to create a render with a lower resolution like 480 or 720p. Is there an option in the config for this?

Thanks!

**Comment by michielst** [comment, marked as answer]
Nevermind, i found you can use the settingsTemplate and set it to a custom render setting profile.

**Reply by pyskaty** [reply]
Hello,
I can't find any documentation for settingsTemplate structure. Is it something that I should configure inside AE and then just pass a name for it? Sorry for noob question :/

**Reply by pyskaty** [reply]
@inlife  maybe you can help me with that. I can't figure how to change render quality/resolution using nexrender. Thanks in advance

**Reply by inlife** [reply]
@pyskaty yeah, that is exactly right. You create/configure the template within AE interface, and then just pass a string with the name.
